File: CSES/1069 - Repetitions.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    string str;
    cin >> str;

    int maxx = 1;
    int sum = 1;

    for(int i = 1 ; i < str.length() ; i++){
        if(str[i] == str[i-1]){
            sum++;
        }else{
            sum = 1;
        }
        maxx = max(maxx, sum);
    }

    cout << maxx;

    return 0;
}

File: CSES/1069 - Repetitions.py
string = input()

saved_char = string[0]
counter = 1
maxx = 1
for i in range(1, len(string)):

    if string[i] == saved_char:
        counter += 1
    else:
        saved_char = string[i]
        counter = 1
    maxx = max(counter, maxx)
print(maxx)